ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/InternalCustomerCenter.kt

```diff
@@ -130,25 +134,52 @@ private fun InternalCustomerCenter(
     onAction: (CustomerCenterAction) -> Unit,
 ) {
     val title = getTitleForState(state)
-    CustomerCenterScaffold(
-        modifier = modifier,
-        title = title,
-        onAction = onAction,
-        navigationButtonType =
-        if (state is CustomerCenterState.Success) {
-            state.navigationButtonType
+
+    val colorScheme: ColorScheme
+    if (state is CustomerCenterState.Success) {
+        val isDark = isSystemInDarkTheme()
+        val appearance: CustomerCenterConfigData.Appearance = state.customerCenterConfigData.appearance
+        val accentColor = appearance.getColorForTheme(isDark) { it.accentColor }
+
+        // Only change background when presenting a promotional offer
+        val backgroundColor = if (state.promotionalOfferData != null) {
+            appearance.getColorForTheme(isDark) { it.backgroundColor }
         } else {
-            CustomerCenterState.NavigationButtonType.CLOSE
-        },
+            null
+        }
+
+        colorScheme = MaterialTheme.colorScheme.copy(
+            primary = accentColor ?: MaterialTheme.colorScheme.primary,
+            background = backgroundColor ?: MaterialTheme.colorScheme.background,
```

Comment:
Hmm depending on the settings of the app, this might not look great (like in our test app) but I guess they can change their theme colors if needed... This might be trickier on the hybrids though.

Reply:
hmmm... I think we would have the same issue in iOS right? The problem is the backup if background color is null?

Reply:
Well, this background is only applied to the promotional offer screen, but not to the other screens right? I believe the problem would be on those screens. And yeah I think the same issue could apply in iOS, depending on their app's settings. Maybe not something to act on right now, but ideally we make sure with the default colors of a new app, the other screens look ok

Reply:
right... maybe adding a modifier? not sure. I am honestly starting to not like making the colors configurable remotely lol

Reply:
That's where I moved to thinking about using the Paywalls V2 editor for this :P

Reply:
Right, same. This is just a way to cover that need for now, but it will be replaced with paywalls v2 editor at some point